wallets/metamask/src/cypress/MetaMask.ts

```diff
@@ -0,0 +1,11 @@
+import { lockPage } from '../selectors'
+import { MetaMaskAbstract } from '../type/MetaMaskAbstract'
+
+// @ts-ignore
```

Comment:
can we refrain from using ```ts-ignore``` and ```any```

Reply:
it will be included in a separated PR as mentioned in the comment right below